tests/lib/snaps/test-snapd-service-try/snap-1.yaml

```diff
@@ -0,0 +1,5 @@
+name: test-snapd-service-try
```

Comment:
Instead of copying over, seems better to have these in separate directories. There are already other cases like this under lib/snaps.